samples: nrf9160: modem_shell: add RTT support (#8554)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

    * MQTT support for ``gnss`` command A-GPS and P-GPS.
    * An application-specific modem fault handler.
      The modem fault handler halts application execution in case of a modem crash.

>     * Support to use SEGGER Real Time Transfer (RTT) instead of UART.

Reviewer: ```suggestion
    * Support for SEGGER's Real Time Transfer (RTT) instead of UART.
```
